docs/protocol/core/v3.0.rst

```diff
@@ -314,82 +359,111 @@ Core data types
      - Numerical type
      - Size (no. bytes)
      - Byte order
-   * - `bool`
-     - Boolean, with False encoded as `\\x00` and True encoded as `\\x01`
+   * - ``bool``
+     - Boolean, with False encoded as ``\\x00`` and True encoded as ``\\x01``
      - 1
      - None
-   * - `i1`
+   * - ``i1``
      - signed integer
      - 1
      - None
-   * - `<i2`
+   * - ``<i2``
      - signed integer
      - 2
      - little-endian
-   * - `<i4`
+   * - ``<i4``
      - signed integer
      - 4
      - little-endian
-   * - `<i8`
+   * - ``<i8``
      - signed integer
      - 8
      - little-endian
-   * - `>i2`
+   * - ``>i2``
      - signed integer
      - 2
      - big-endian
-   * - `>i4`
+   * - ``>i4``
      - signed integer
      - 4
      - big-endian
-   * - `>i8`
+   * - ``>i8``
      - signed integer
      - 8
      - big-endian
-   * - `u1`
+   * - ``u1``
      - signed integer
```

Comment:
unsigned integer?

Reply:
probably my fault when I typed a vim macro. 

I gave you access to my fork if you want to make changes directly.